test/common/testlib.py

```diff
@@ -113,7 +113,7 @@ def attach(filename, move=False):
 class Browser:
     def __init__(self, address, label, machine, pixels_label=None, port=None):
         if ":" in address:
-            (self.address, unused, self.port) = address.rpartition(":")
+            self.address, _, self.port = address.rpartition(":")
```

Comment:
It feels a little silly to use the API that returns the separator, and then ignoring it. That's pretty much exactly what `rsplit(':', 1) is used for.

Reply:
I think that's a matter of taste.  I always use partition for this stuff...